app/src/main/java/com/hover/stax/settings/PinsViewModel.java

```diff
@@ -81,14 +81,14 @@ public void removeAccount(Channel channel) {
 		channel.selected = false;
 		channel.defaultAccount = false;
 		repo.update(channel);
-		if (channels.getValue() == null || !changeDefault) return;
+		/*if (channels.getValue() == null || !changeDefault) return;
```

Comment:
Why is this commented?

Reply:
I've uncommented it back. It wasn't removing the last account. I understand it's trying to set the last standing account as the default account; so tagging it as a bug and added a TODO to the latest commit to fix this my morning tomorrow.